### ISSUE TITLE
vscode update to 1.16, info background is wrong

### ISSUE THREAD

**Issue by n1ngx** [akamud/vscode-theme-onedark#71]
default theme is ok,
but onedark is red

**Comment by akamud** [comment]
Something changed in the new version. Thanks for letting me know, will fix this in the new version and I'll let you know.

**Comment by akamud** [comment]
Found the problem, an innocent missing `#`. I'll release a new version tomorrow.

In the meantime, you can [customize](https://code.visualstudio.com/docs/getstarted/themes#_customize-a-color-theme) it via:

```json
"workbench.colorCustomizations": {
		 "notification.background": "#21252B"
}
```

**Comment by akamud** [comment]
Fixed in 2.0.4.
